Mathlib/Topology/Algebra/Polynomial.lean

```diff
@@ -165,8 +165,7 @@ theorem coeff_le_of_roots_le {p : F[X]} {f : F →+* K} {B : ℝ} (i : ℕ) (h1
   obtain hB | hB := lt_or_le B 0
   · rw [eq_one_of_roots_le hB h1 h2 h3, Polynomial.map_one, natDegree_one, zero_tsub, pow_zero,
       one_mul, coeff_one]
-    split_ifs <;> norm_num [h]
-    simp [‹0 = i›]
+    split_ifs with h <;> simp [h]
```

Comment:
this will conflict with #6600

Reply:
In a good way though: we're both proposing the same thing. I wonder if git is even smart enough enough that would not conflict.

Reply:
I actually don't know, maybe it will?